urlless/.gitignore

```diff
@@ -0,0 +1,34 @@
+# built application files
```

Comment:
One single .gitignore for the whole project should be enough. Keep it in the project root.